templates/archives/stories/meesue_story.jinja2

```diff
@@ -0,0 +1,57 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+The Rise and Fall of Project Whirlwind
+{% endblock %}
+
+{% block story_credit %}
+
+{% endblock %}
+
+
+{% block story_body %}
+<p>
+    Arguably the first large-scale and high-speed computer, the development of the Whirlwind Project started in 1944 by
+    the MIT Servomechanisms Laboratory in a project directed by Jay Forrester for the United States Navy. This project
+    was initially meant to design an aircraft simulator during World War II, but researchers soon realized that the
+    system was too slow and inaccurate. As a result of this, the simulation did not give a realistic enough feel for it
+    to be used as a proper training method for pilots. A key part of this project was the replacement of electrostatic
+    storage tubes with the
+    <a href="https://infinitehistory.mit.edu/video/edward-murrows-see-it-now%E2%80%94jay-forrester-and-whirlwind-computer-1951">
+        magnetic core memory,
+    </a>
+    which allowed for high speeds and reliability, just not enough for the computer to function properly as a
+    universal flight simulator. Because of the use of the magnetic core memory, Whirlwind became a fully functioning
+    computer that was used for scientific and military research.
+</p>
+
+<figure class="figure float-right">
+    <img src="{{ static('img/stories/meesue_story.jpg') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="The Whirlwind Project" />
+    <figcaption class="figure-caption">The Whirlwind Project</figcaption>
+</figure>
+
+<p>
+    The development of Whirlwind served as a milestone in the history of computation at MIT, but it all came at a steep
+    cost. The expected cost of the project was initially $200,000, and it was projected to take approximately two years
+    to finish. Although with any project there is always the inevitability of going over budget and taking more time
+    than expected, the Whirlwind Project went over the projections far more than other computer projects of that time.
+</p>
+
+<p>
+    Starting in 1947, the Whirlwind Project received funding from the Office of Naval Research (ONR). A huge problem
+    with this project was the sheer amount of resources going into it. The research team consisted of more than one
+    hundred people, and the budget was close to $3 million. This alone took up 20% of the ONR’s total research budget.
```

Comment:
Some of your colleagues have mentioned what the money at the time is now worth in today's money, which would be worth pointing out so people could better understand the magnitude of investment.